deploy/values.yml

```diff
@@ -40,8 +40,8 @@ resources:
   limits:
     memory: 256Mi
```

Comment:
We need to discuss the "rule" again, I don't necessarily agree with it. It makes the limit too low, and if an app runs out of mem it crashes